build.rs

```diff
@@ -39,6 +39,7 @@ fn get_git_hash() -> Option<String> {
     if let Ok(output) = Command::new("jj")
         .args([
             "--ignore-working-copy",
+            "--color=none",
```

Comment:
`--color=never`?

Reply:
That's right, never ever. Thank you!

This worked by accident. At some point, we should become more strict about invalid option values.